### PR TITLE
Sprint 5

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/
+build/*
+build_test/*
+/.vscode/
+build/CMakeFiles/arduino-template.elf.dir/build.make
```
Comment (JulianvDoorn) [inline .gitignore]: Redundant, since build/ should be ignored as a whole

File: src/main.cpp [hunk 2]
```diff
@@ -1,10 +1,12 @@
+#include "moving_platform.hpp"
 #include "wrap-hwlib.hpp"
 
 int main() {
     WDT->WDT_MR = WDT_MR_WDDIS;
 
     hwlib::wait_ms(1000);
-    hwlib::cout << "Hello world!" << hwlib::endl;
+    MovingPlatform::Platform test;
+    hwlib::cout << "Hello world!" << test.getWheelSize() << hwlib::endl;
```
Comment (JulianvDoorn) [inline src/main.cpp]: "Hello world!" is not part of the vertical slice, I suppose

File: src/moving_platform.hpp [hunk 3]
```diff
@@ -0,0 +1,106 @@
+/**
+ * @file moving_platform.hpp
+ * @brief     Moving platform class
+ * @author    Olivier verwoerd, Maarten Wassenaar, Daniel van Vliet & Jasper Smienk.
+ * @license   See LICENSE
+ */
+#include <stdint.h>
```
Comment (JulianvDoorn) [inline src/moving_platform.hpp]: hwlib provides int16_t types as well. If this module is compiled for Arduino it is advised to just include hwlib.

**Review by JulianvDoorn — CHANGES_REQUESTED** [review]
Code looks good. Good Doxygen coverage. Minor details were commented during this review.

**Comment by ghost** [comment]
@JulianvDoorn Everything you requested has been updated.

**Review by JulianvDoorn — APPROVED** [review]

**Pull request merged**